ts/screens/wallet/FirstTransactionSummaryScreen.tsx

```diff
@@ -0,0 +1,245 @@
+/**
+ * This screen shows the transaction details.
+ * It should occur after the transaction identification by qr scanner or manual procedure.
+ * TODO:
+ * - integrate contextual help
+ *    https://www.pivotaltracker.com/n/projects/2048617/stories/158108270
+ *
+ * - check availability of displayed data. Define optional data and implement their rendering as preferred
+ *
+ *
+ */
+
+import {
+  Body,
+  Button,
+  Container,
+  Content,
+  H1,
+  H3,
+  Icon,
+  Left,
+  Text,
+  View
+} from "native-base";
+import * as React from "react";
+import { Image, StyleSheet } from "react-native";
+import { Col, Grid, Row } from "react-native-easy-grid";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import { WalletAPI } from "../../api/wallet/wallet-api";
+import { WalletStyles } from "../../components/styles/wallet";
+import AppHeader from "../../components/ui/AppHeader";
+import PaymentSummaryComponent from "../../components/wallet/paymentSummary/PaymentSummaryComponent";
+import UpdatedPaymentSummaryComponent from "../../components/wallet/paymentSummary/UpdatedPaymentSummaryComponent";
+import I18n from "../../i18n";
+import variables from "../../theme/variables";
+import {
+  NotifiedTransaction,
+  TransactionEntity,
+  TransactionSubject
+} from "../../types/wallet";
+
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+const styles = StyleSheet.create({
+  padded: {
```

Comment:
this should be a top level utility (ie. `paddedContent`)

Reply:
it is not on Grid component but on Content component. The screen which hosts this Grid should be not affected by the padding on the Content component to display full width background color.

Reply:
I don't have the context, but what I meant is that when we have some properties that comes together multiple times


there may be some top level variable / utility between the theme variables, ie.

```ts
paddingX: {
  paddingRight: variables.contentPadding,
  paddingLeft: variables.contentPadding
}
```

that can be reused like:

{ ...paddingX, <other style props> }

Reply:
ok, thank you, now it is more clear. I used this approach in many PRs. Maybe it could be a  little story which adapt it for all the screens